test/unit/app/helpers/admin/edition_actions_helper_test.rb

```diff
@@ -2,12 +2,14 @@
 
 class Admin::EditionActionsHelperTest < ActionView::TestCase
   setup do
-    @editions = ["Case studies",
-                 "Calls for evidence",
+    ConfigurableDocumentType.setup_test_types(build_configurable_document_type("history_pages", { "title" => "History page", "settings" => { "publishing_api_schema_name" => "history", "translations_enabled" => false } }))
```

Comment:
Everywhere else in the tests, we've gone with the singular - not that it matters too much. And we can also drop the `translations_enabled` property, since that's not the subject of the test:

```suggestion
    ConfigurableDocumentType.setup_test_types(build_configurable_document_type("history_page", { "title" => "History page", "settings" => { "publishing_api_schema_name" => "history" } }))
```